book/src/user/supported-platforms.md

```diff
@@ -0,0 +1,48 @@
+# Platform Support
+
+Support for different platforms ("targets") are organized into three tiers,
+each with a different set of guarantees. For more information on the policies
+for targets at each tier, see the [Target Tier Policy](target-tier-policy.md).
+
+Targets are identified by their "target triple", which is the string to inform
+the compiler what kind of output should be produced.
+
+## Tier 1
+
+Tier 1 targets can be thought of as "guaranteed to work". The Zebra project
+builds official binary releases for each tier 1 target, and automated testing
+ensures that each tier 1 target builds and passes tests after each change.
+
+For the full requirements, see [Tier 1 target policy](target-tier-policy.md#tier-1-target-policy) in the Target Tier Policy.
+
+target | notes
```

Comment:
We're missing documentation about what kinds of binaries we're building and distributing for each Tier 1 platform.
Can we add another table that lists the distribution methods?

Currently that's just Docker.